<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool</class>
 <widget class="QMainWindow" name="Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASR Post Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="playerLayout">
        <item>
         <widget class="VideoWidget" name="m_videoWidget" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="slider_position">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_position">
            <property name="text">
             <string>00:00 / 00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PlayerControls" name="m_playerControls" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="editorLayout">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="Editor" name="m_editor"/>
       <widget class="TextEditor" name="m_wordEditor" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="close"/>
   </widget>
   <widget class="QMenu" name="menuMedia_Player">
    <property name="title">
     <string>Player</string>
    </property>
    <addaction name="player_open"/>
    <addaction name="player_togglePlay"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="editor_openTranscript"/>
    <addaction name="editor_save"/>
    <addaction name="editor_debugBlocks"/>
    <addaction name="editor_jumpToLine"/>
    <addaction name="editor_splitLine"/>
    <addaction name="editor_mergeUp"/>
    <addaction name="editor_mergeDown"/>
    <addaction name="editor_findReplace"/>
    <addaction name="editor_toggleWords"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMedia_Player"/>
   <addaction name="menuEditor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="player_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="editor_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="editor_save">
   <property name="text">
    <string>Save Transcript</string>
   </property>
  </action>
  <action name="editor_openTranscript">
   <property name="text">
    <string>Open Transcript</string>
   </property>
  </action>
  <action name="editor_debugBlocks">
   <property name="text">
    <string>Debug Blocks</string>
   </property>
  </action>
  <action name="editor_jumpToLine">
   <property name="text">
    <string>Jump to Highlighted Line</string>
   </property>
  </action>
  <action name="player_togglePlay">
   <property name="text">
    <string>Play / Pause</string>
   </property>
  </action>
  <action name="editor_splitLine">
   <property name="text">
    <string>Split Line</string>
   </property>
  </action>
  <action name="editor_mergeUp">
   <property name="text">
    <string>Merge Up</string>
   </property>
  </action>
  <action name="editor_mergeDown">
   <property name="text">
    <string>Merge Down</string>
   </property>
  </action>
  <action name="editor_findReplace">
   <property name="text">
    <string>Find Replace</string>
   </property>
  </action>
  <action name="editor_toggleWords">
   <property name="text">
    <string>Toggle Word Editor</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayerControls</class>
   <extends>QWidget</extends>
   <header>playercontrols.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Editor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor.h</header>
  </customwidget>
  <customwidget>
   <class>TextEditor</class>
   <extends>QWidget</extends>
   <header>texteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
